dev/scripts/src/alluxio.org/build-distribution/cmd/generate-fuse-tarball.go

```diff
@@ -22,8 +22,8 @@ func Fuse(args []string) error {
 	// flags
 	addCommonFlags(fuseCmd, &FlagsOpts{
 		TargetName: fmt.Sprintf("alluxio-fuse-%v.tar.gz", versionMarker),
-		UfsModules: strings.Join(defaultModules(fuseUfsModules), ","),
```

Comment:
wrapping the `defaultModules()` means that you would not be including the hadoop-3.3 module since it's default flag is false. i assume that is the behavior you want

Reply:
change to use hadoop-3.2 instead of 3.3 since it looks like common (it's marked as default)